monitoring-and-logging.md

```diff
@@ -0,0 +1,142 @@
+
+    ![Ceph Percent Used](img/acceptance-tests/ceph-percent-used.png)
+
+## MultiClusterObservability documentation
+
+Here are some useful links to the MultiClusterObservability documentation:
+
+- [APIs Red Hat Advanced Cluster Management for Kubernetes 2.5](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html/apis/apis#rhacm-docs_apis_multiclusterobservability_jsonmulticlusterobservability_observabilityadvanced)
+- [Observing environments introduction Red Hat Advanced Cluster Management for Kubernetes 2.5](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.5/html/observability/observing-environments-intro#enabling-observability)
+- [Managing applications Red Hat Advanced Cluster Management for Kubernetes 2.0](https://access.redhat.com/documentation/en-us/red_hat_advanced_cluster_management_for_kubernetes/2.0/html/manage_applications/managing-applications)
+
+
+# Logging
+
+## Logging Operators
+
+- Logging in the cluster is provided by the Red Hat [Red Hat OpenShift Logging here](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/k8s/ns/openshift-logging/operators.coreos.com~v1alpha1~ClusterServiceVersion/cluster-logging.5.5.0).
+- We combine the OpenShift Logging Operator with the [Loki Operator here](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/k8s/ns/openshift-operators-redhat/operators.coreos.com~v1alpha1~Subscription/loki-operator), so that the Logging Operator sends the infrastructure, audit, and application logs to the Loki Operator where they are stored in an Object Bucket.
+- The OpenShift Logging Operator has a dependency on the [Elasticsearch Operator here](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/k8s/ns/openshift-operators-redhat/operators.coreos.com~v1alpha1~Subscription/elasticsearch-operator). Whether you use Elasticsearch for storing logs or using Loki, you still need the Elasticsearch Operator installed for required dependent CustomResourceDefinitions.
+
+### Loki operator
+
+- The Loki Operator allows you to set up LokiStacks, AlertingRules, RecordingRules, and RulerConfigs based on your cluster logs for infrastructure, audit, and applications. See the [ Loki Operator here ](https://console-openshift-console.apps.nerc-ocp-infra.rc.fas.harvard.edu/k8s/ns/openshift-operators-redhat/operators.coreos.com~v1alpha1~ClusterServiceVersion/loki-operator.5.4.6)
```

Comment:
another 404.  5.5.5 vs 5.4.6 I think

Reply:
Thanks @aabaris , I have updated the link.